ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/_cq_dialog/.content.xml

```diff
@@ -60,30 +61,30 @@
                                     sling:orderBefore="guideGlobalLazyField"
                                     sling:resourceType="granite/ui/components/coral/foundation/form/datepicker"
                                     fieldDescription="Please enter the date in the required format  &quot;yyyy-mm-dd&quot;."
-                                    fieldLabel="Minimum Value"
+                                    fieldLabel="Minimum date / Start date"
```

Comment:
Start date has a different meaning - `Minimum date` looks better in itself. Same for End date.